draft-bdmgct-spring-srv6-security.md

```diff
@@ -228,8 +199,35 @@ Following the spirit of [RFC8402], the current document  mandates a filtering me
 
 It should be noted that in some threat models the distinction between internal and external attackers depends on whether an attacker has access to a cryptographically secured (encrypted or authenticated) domain. Specifically, in some of these models there is a distinction between an attacker who becomes internal by having physical access, for example by plugging into an active port of a network device, and an attacker who has full access to a legitimate network node, including for example encryption keys if the network is encrypted. The current model does not distinguish between these two types of attackers and there is no assumption about whether the SR domain is cryptographically secured or not. Thus, some of the attacks that are described in the next section can be mitigated by cryptographic means, as further discussed in {{hmac}}.
 
+# Impact
+
+One of the important aspects of a threat analysis is the potential impact of each threat. SRv6 allows for the sending of IPv6 packets via arbitrary paths. An attack on SRv6 may cause packets to traverse arbitrary paths within an SR domain. This may allow an attacker to perform a number of attacks on the victim networks and hosts that would be mostly unfeasible for a non-SRv6 environment.
+
+The threat model in [ANSI-Sec] classifies threats according to their potential impact, defining six categories. For each of these categories we briefly discuss its applicability to SRv6 attacks.
```

Comment:
I believe there are a number of aspects that need to be addressed here:

1. Things like STRIDE, ANSI-sec, or the like, may be approaches for conducting threat modeling -- i.e., finding possible threats. However, they don't really classify the impact, because the impact may be affected by a number of aspects, including mitigating factors or techniques.

2. The method/technique one uses simply has the goal of finding threats. As such, i wouldn't flag any threats as "out of scope" -- they actually aren't. If anything, we might be unable to find any. 
3. Rather than listing the "threats" in ANSI-sec, I would probably add a top level "threats" section, and then add one level-2 section for each of the threts, where we can elaborate how each of "unauthorized access", "masquerade" (etc) could be launched/implemented in a SRV6 domain.
i.e., rather than focus on the model in ANSI-sec, just I'd just make use of it. -- and if we were to find a threat that doesn;t fit cleanly in the taxonomy in ANSI-sec,  nevertheless add it as a level-2 section (child of top level "Threats" section), together with the other ones. -- IOW, any threat-modeling framework or methodology are simply tools to find threats, as opposed to e.g., an exhaustive/specific taxonomy for threats.